### PR TITLE
Add astropy affiliated package linetools

### PR DESCRIPTION
**Pull request by mwcraig**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by mwcraig** [comment]
@profxj @ntejos -- would you like to be added as maintainers of this conda recipe?

**Comment by profxj** [comment]
yes, that would be accurate.  Thanks.

File: recipes/linetools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 30af47d42933f6e2b3494ab1a74156ae920898bbba2ade48f15b658e109e8ee0
+  patches:
+    - setup.cfg.patch
+
+build:
+  number: 0
```
Comment (chrisburr) [inline recipes/linetools/meta.yaml]: I think this could be made `noarch`?
```suggestion
  number: 0
  noarch: python
```

Comment (mwcraig) [reply recipes/linetools/meta.yaml]: I had hoped to do that, but [the setup call has a scripts argument](https://github.com/linetools/linetools/blob/master/setup.py#L129) and the documentation say [that precludes `noarch`](http://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python).

If the documentation are incorrect I'm happy to make it `noarch`.

Comment (chrisburr) [reply recipes/linetools/meta.yaml]: Ah yes you're right, LGTM 👍

**Pull request merged**